setup.sh

```diff
@@ -1,2 +1,15 @@
 #!/usr/bin/env bash
-brew install python3
\ No newline at end of file
+
+
+if [ -z "$(which python3)" ];
+then
+    echo "install"
+    brew install python3
+else
+    echo "update"
+    brew upgrade python3
```

Comment:
Hmm - I'm not sure about installing/upgrading python without warnings.  When do people run setup.sh?  I'd prefer something that fails if python is not installed or is too old, with a warning "please run brew install..." - if that's not too much more work.

Reply:
Yeah, Vanessa and I spent a while chatting about this, and didn't come up with anything that we thought was perfect.

I prefer your suggestion though, so I'll change it to that in a new PR.